ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -69,6 +69,12 @@ tests:
   container:
     from: src
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: aws-e2e-ccm-install
```

Comment:
No harm in adding skip_if_only_changed field same as other jobs, would be useful when the job is functional.

Reply:
Done!